src/components/LegislativeProcess/VotingProcessImageCircle/VotingProcessImageCircle.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+	export let src = '';
+	export let alt = '';
+	export let title: string;
+	export let description = '';
+</script>
+
+<div class="flex-1 max-w-[280px]">
+	<div class="mb-2 flex justify-center text-center">
+		{#if src}
+			<img class="w-32 h-32 rounded-full" {src} alt={alt ? `${alt}` : `${title}`} />
+		{:else}
+			<div class="bg-ui-01 w-32 h-32 rounded-full flex justify-center items-center">
```

Comment:
To make adding images easier later on. Can you add an `image` prop that received the image URL and put it in `<img />` here? Then you can add a placeholder image URL as a prop value from https://placehold.co/

Reply:
That mean no if-else block and set default value of `src` prop from [https://placehold.co/](https://placehold.co/)

it will look like image below , right ? 
or did you mean I need to change prop name from `src` to `image` ? 

![image](https://github.com/wevisdemo/parliament-watch/assets/110652592/cbe4eb58-81ca-4da6-a97e-50f389d82fc1)

Reply:
Ah yes, somehow I didn't see the `src` prop 🤦‍♂️. Yes, you just get rid of `if-else`.

We better put the mock src value from https://placehold.co/ outside the component, instead of using the default prop. Even though the mock src will be duplicated, it is temporary. When we have the final graphic, we can just replace the `src` prop from the outside without removing the default prop that we are not going to use anymore.